### DISCUSSION TITLE
Error to deploy in main branch

### DISCUSSION THREAD

**Discussion by pakolkf** [symfony/symfony#45138] [Q&A]
Hi!! I'm trying to push a development to the main branch of maker-bundle on github and it's impossible for me since it doesn't pass the tests. I have seen that this error has been occurring for some time. The error is as follows: "To use Doctrine entity attributes, you will need PHP 8, doctrine/orm 2.9, doctrine/doctrine-bundle 2.4, and symfony/framework-bundle 5.2." Any ideas?

**Comment by emmanuellemc** [comment]
I have exactly the same error, except it happens when I try to create a new Entity. It's on a brand new Symfony project, I have double checked and everything is up to date so for me it makes no sense... Anyone else have came across this and hopefully solved the issue ?

**Reply by fd6130** [reply]
Is it a light weight version of Symfony project or Full version of Symfony project? If light weight version, perhaps you just forgot to install doctrine?

**Reply by emmanuellemc** [reply]
Full, I followed the documentation (symfony new my_project_directory --webapp) and I installed doctrine as well following the recommandations. 

EDIT : I guess I found the source of the error ! Even after running composer updates on framework-bundles (which returns a "up to date" message), I get this as my configuration : 
```
composer recipes symfony/framework-bundle
name             : symfony/framework-bundle
version          : 4.4
status           : up to date
```
What I did > At first I tried to change the require to 5.2 in my composer.json but it failed because of conflicts of requires. Turned out almost everything was locked to 4.4 version. So I manually switched to "^5.0." all the symfony requires and ran a `composer update "symfony/*"` Now everything works like a charm ! 
@pakolkf please check if everything is up to date on your side and check for any locks it's likely to be the source of your problem :)
